app/javascript/components/App.jsx

```diff
@@ -0,0 +1,49 @@
+import ReactDOM from 'react-dom';
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+  Routes,
+} from 'react-router-dom';
+
+import { Provider } from 'react-redux';
+import Message from './message.jsx';
+import configureStore from '../configureStore.js';
+
+const store = configureStore();
+
+export default function App() {
```

Comment:
- [ ] I will suggest you rename the `App.jsx` to `App.js`. Components in reacts always have the `.js` extensions.

Reply:
Hey @enaburekhan You are saying: I will suggest you rename the App.jsx to App.js. ❌ ❌ ❌  
Kindly check DHH video to know why you are **wrong** about your suggestion 
[React app to use esbuild with JSX in Rails 7](https://www.youtube.com/watch?v=Chiu-0EVW3g)

Reply:
@Bishoy-Samwel, thank you for the update. I watch the video. And I have just learned something new. 
As a code reviewer, I will first run the app to ensure it's working as expected. The react section was not working with the rails when I did the review.
Hence the need for these changes.  Thanks for this new information. But I think the project requirements is still centered around using a `bundler/transpiler`. So we can make do with it until the microverse management decides to upgrade the project requirement to use `importmap`.

Reply:
@enaburekhan Hey, Thanks for your time and your reply, 

 > But I think the project requirements is still centered around using a bundler/transpiler
That is correct  it has a bundler of course there is [`jsbundling-rails`](https://github.com/rails/jsbundling-rails) gem and the bundler is `esbuild` instead of `webpack` 
 >   So we can make do with it until the microverse management decides to upgrade the project requirement to use importmap.
We don't use `importmap` but `jsbundling-rails` and `esbuild`
 >  Lastly, It's always better to refer one to the official documentation in a case like this.
He is DAVID HEINEMEIER HANSSON  the creator of the popular Ruby on Rails